src/include/common/int128_t.h

```diff
@@ -0,0 +1,83 @@
+#pragma once
+
+#include <cstdint>
+
+#include "common/api.h"
+#include "common/string_utils.h"
+
+namespace kuzu::common {
+
+struct KUZU_API int128_t {
+    uint64_t low;
+    int64_t high;
+
+    int128_t() = default;
+    int128_t(int64_t value); // NOLINT: Allow implicit conversion from `int64_t`
```

Comment:
BTW: Where are we going to use this implicit cast?

Reply:
`createDefaultValue` in value.cpp 
```
case LogicalTypeID::INT128:
        return Value(int128_t(0));
```